docs/faq.md

```diff
@@ -0,0 +1,261 @@
+ 
+### How do I handle cancellations, changes and post-booking services for bookings made with Flight Create Orders in Self-Service?
+
+For Self-Service users, all post-booking services must be handled offline with the consolidator you work with for ticket issuance. In general, these actions can be made while the Passenger Name Record (PNR) is queued for ticketing (before the ticket is issued), though their availability once a ticket has issued depends largely on the consolidator and the clauses of your agreement. 
+
+### How do I handle refunds for flights booked with Flight Create Orders in Self-Service?
+
+Refunds must be handled offline directly with your consolidator. 
+
+## Technical support
+
+### What kind of support does Amadeus for Developers offer?
+
+There are two different support paths available based on our two different offer: Self-Service and Enterprise.
+
+1. Self-Service users have at their disposal detailed documentation, guides and SDKs, to help them solve any doubts they may have. Check the Self-Service Docs page for more information. For any other Self-Service support queries, such as billing issues or a refund request, please go to the support section and click on contact us about Self-Service support.
+2. Enterprise users have access to dedicated support. If you are an Enterprise user, get in touch with your Account Manager or open a ticket via the Amadeus Service Hub.
+ 
+### Where do I go for Self-Service technical support? What does it cost?
+
+If you are a Self-Service customer experiencing a technical issue, you should do the following:
+ 
+1. First, look for an answer in our Self-Service APIs Docs and this FAQs page. We update this page regularly with explanations on fixing common issues.
+2.  Search for a solution in Stack Overflow, or ask the community for help. Our developer advocacy team actively monitors and answers the questions on Stack Overflow that relate to our APIs.
+3. Finally, if you are still experiencing a problem with Self-Service APIs, you can get in touch with our developer advocates via the contact form in the Support page. We will try to get back to you as quickly as possible, however please understand that in times of high demand we may not be able to guarantee a prompt answer.
+
+### Do you offer phone support for Self-Service APIs?
+
+We do not currently offer phone support for Self-Service APIs. If you need assistance you can get in touch with our Developer Advocates via the contact form in the Support page. Please keep in mind that in times of high demand we may not be able to guarantee a prompt answer.
+
+### How can I report bugs or suggest improvements to the Self-Service section?
+
+We love feedback from our community and it helps us create the best possible product for all users! If you want to report a bug or suggest improvements, please go to the Support section and get in touch using the Contact form.
+
+## Booking code
+
+### What exactly is the booking code?
```

Comment:
this one feels a bit out of the blue xD Is it for hotels only or flights too?

Reply:
discussed with Mat, we can remove it